.drone.yml

```diff
@@ -0,0 +1,28 @@
+clone:
+  git:
+    image: plugins/git
+    depth: 50
+    tags: true
+
+pipeline:
+  npm-install:
+    group: install
+    image: node:10-alpine
+    commands: ["npm install"]
+
+  eslint:
+    group: install
+    image: marcbachmann/eslint:4.19.1
+    secrets: [gh_token]
+
+  test:
+    group: test
+    image: node:10-alpine
+    commands: ["npm run test -s"]
```

Comment:
silent, right? I think good for now 👍

Reply:
yes, that's the default I'm using in all the droneci configs. Silent just means that it won't print the  npm info at all. So it won't show the command that gets executed and also not the stack trace if it fails. It just shows the raw stdout and stderr of the child process.